code/api/src/modules/user/mutations.js

```diff
@@ -37,4 +37,19 @@ export const userRemove = {
     }
   },
   resolve: remove
-}
\ No newline at end of file
+}
+
```

Comment:
The mutation below is what allows one to edit a user's email. You have to provide an ID (so GraphQL can find which user), then the new email.

Reply:
Is email the only thing we can edit with this mutation?

Reply:
@EHale64 yeah